cmd/tenant_controller/main.go

```diff
@@ -32,6 +36,18 @@ func init() {
 }
 
 func main() {
+
+	go func() {
+		sigs := make(chan os.Signal, 1)
+		signal.Notify(sigs, syscall.SIGQUIT)
+		buf := make([]byte, 1<<20)
```

Comment:
The buffer size is not easily understandable.

Reply:
Now provided as simple integer value.